src/components/faceVerification/standalone/components/GiveUpButton.js

```diff
@@ -15,11 +15,17 @@ const GiveUpButton = () => {
   const onReasonChosen = useCallback(
     (reason = undefined) => {
       const data = pickBy({ reason }, negate(isUndefined))
-
+      const redirect = () => fvRedirect(false, reason)
+      
       fireEvent(FV_GIVEUP, data)
 
       // await before analytics scripts will perform some activity
-      window.requestIdleCallback(() => fvRedirect(false, reason))
+
+      if (window.requestIdleCallback) {
+        window.requestIdleCallback(redirect)
+      } else {
+        window.requestAnimationFrame(redirect)
```

Comment:
in the case it does not work just use setTimeout:

```suggestion
        setTimeout(redirect, 0)
```

Reply:
also if 0 value not work try 20-50ms delay

Reply:
no initial suggestion of requestAnimationFrame works